app/views/denounces/_form.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= javascript_pack_tag 'addressSelectors.js' %>
+<%= bootstrap_form_with(model: denounce, layout: :horizontal, label_col: "col-12", control_col: "col-12", inline_errors: true) do |form| %>
+<div class="card-body mx-4 my-4">
+  <div class="row">
+    <div class="col-12 col-lg-6">
+      <h3>Denounce Information</h3>x
+      <%= form.datetime_select :event_date %>
+      <%= form.select :area, Denounce.areas.keys %>
+      <%= form.text_field :title %>
+      <%= form.text_field :description %>
+      <%= form.check_box :anonymouse, label: "Make it anonymous" %>
+    </div>
+
+    <div class="col-12 col-lg-6">
+      <h3>Address</h3>
+      <%= form.fields_for :address, @address do |form_field| %>
+      <div class="row">
+        <div class="col-lg-6"><%= form_field.text_field :street, label: "Street" %></div>
+        <div class="col-lg-4"><%= form_field.text_field :number, label: "Number" %></div>
+      </div>
+      <div class="row">
+        <div class="col-lg-5">
+          <%= form_field.select :state,
+                    options_for_state_select,
+                    { include_blank: true },
+                    { class: "statesSelector", required: true }
+                  %>
```

Comment:
I think could be better use a `collection_select` here https://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-collection_select

Reply:
marking it as resolved for the same reason as above. We are going to implement this suggestion in a smaller refactoring branch.